app/javascript/admin/controllers/error_controller.js

```diff
@@ -0,0 +1,12 @@
+import { Controller } from 'stimulus';
```

Comment:
Created this `error_controller` to globalize the ability to generate error alerts in Admin. For async call, these error alerts will replace the `flash[:error]` that is generated by Rails server-side.